examples/simultaneous_translation/docs/evaluation.md

```diff
@@ -96,17 +103,63 @@ class Agent(object):
 
  
 ```
-Here are the implementations of agents for [text *wait-k* model](../eval/agents/simul_trans_text_agent.py) and [speech *wait-k* model](../eval/agents/simul_trans_speech_agent.py).
 
-## Quality
-The quality is measured by detokenized BLEU. So make sure that the predicted words sent to server are detokenized. An implementation is can be find [here](../eval/agent.py)
-=======
+Here is an exmaple of a customized agent. 
+First of all, a name needs to be registered. 
+Next, reload the agent functions according to the translation model. 
```

Comment:
What do you mean by reload here?

Reply:
It means reload the python function, do you prefer another expression?